.travis.yml

```diff
@@ -54,47 +44,38 @@ jobs:
   allow_failures:
     - php: nightly
     - env: DB=pgsql POSTGRESQL_VERSION=11.0
-
-  exclude:
-    - php: 7.1
-      env: DB=sqlite
-    - php: 7.1
-      env: DB=mysql
-    - php: 7.1
-      env: DB=mysqli
+    - php: 7.3
+      env: DB=ibm_db2
 
   include:
-    - stage: Test
-      php: 7.1
-      env: DB=sqlite COVERAGE=yes
-
-    - stage: Test
-      php: 7.1
-      env: DB=mysql COVERAGE=yes
-
-    - stage: Test
-      php: 7.1
-      env: DB=mysqli COVERAGE=yes
-
+# BEGIN TEST BUILD MATRIX - DO NOT EDIT BELOW ▼▼▼ USE THE TRAVIS MATRIX BUILDER SCRIPT
     - stage: Test
       php: 7.1
       env: DB=mysql MYSQL_VERSION=5.7 COVERAGE=yes
```

Comment:
Would it make sense to collect coverage on the latest stable PHP version where we expect all drivers to be tested against all platform versions?

Reply:
It does indeed! Changed to 7.2.